Src/WitsmlExplorer.Api/Workers/BaseWorker.cs

```diff
@@ -5,13 +5,14 @@
 using Microsoft.Extensions.Logging;
 
 using WitsmlExplorer.Api.Extensions;
+using WitsmlExplorer.Api.Jobs;
```

Comment:
namespace not used

Reply:
It is actually used in the following PR #1418 so I think it can remain here 🙃